### PR TITLE
Enable console for installers and improve docs

### PR DESCRIPTION
**Pull request by giggsoff**
Let's enable console in installers and add few words about enabling of console.

### CODE REVIEW

**Review by deitch — APPROVED** [review]
So this is only during the install phase, and not any later phase? Sounds good.

Thank you especially for documenting it.

**Comment by rouming** [comment]
Would be perfect to have a console enabled by default for debug builds! I made this wish for a New Year :)

**Comment by giggsoff** [comment]
> Would be perfect to have a console enabled by default for debug builds! I made this wish for a New Year :)

Well, some kind of this was open for a while here https://github.com/lf-edge/eve/pull/2975. But during discussion with @eriknordmark and @mikem-zed we decided that we should find better way to compose "debug builds", e.g. using conposition of yml templates or so. Before stabilization of such approach we should at least have doccumentation, which I added here.

**Comment by rouming** [comment]
As I said, this is just a wish, and wishes not always come true:)

**Review by eriknordmark — APPROVED** [review]
LGTM

File: docs/BOOTING.md [hunk 1]
```diff
@@ -265,3 +265,10 @@ You need to extract needed files with something like `docker run lfedge/eve:late
 You will see a set of files in the current directory to locate into you tftp server to boot Raspberry from it. Also, you should set dhcp-boot option of your
 dhcp server to `ipxe.efi` (actually, it will use configuration from `ipxe.efi.cfg`). Files `kernel`, `initrd.img` and `initrd.bits`
 should be available via HTTP/HTTPs and you need to modify `ipxe.efi.cfg` with location of those files.
+
+## Console access
+
+Access via console enabled during initial bootstrap and will be disabled after first reboot of onboarded edge node.
```
Comment (rene) [inline docs/BOOTING.md:271]: "Access via console **is** enabled during initial..."

**Comment by rene** [comment]
> As I said, this is just a wish, and wishes not always come true:)

+1 for the wishes about have console enabled for debug builds...

**Review by eriknordmark — APPROVED** [review]
Re-run eden

**Pull request merged**